Snippets/Snippets_5/Features/FeatureConfiguration.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Snippets5.Features
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Config;
+    using NServiceBus.Features;
+    using NServiceBus.Pipeline;
+
+    class SecondLevelRetries : Feature
+    {
+
+#region FeatureConfiguration
+        public SecondLevelRetries()
+        {
+            EnableByDefault();
+
+            DependsOn<DelayedDeliveryFeature>();
+
+            Prerequisite(context => !context.Settings.GetOrDefault<bool>("Endpoint.SendOnly"), "Send only endpoints can't use SLR since it requires receive capabilities");
```

Comment:
actually that message is copied from the productive code :D I think it's ok to keep that message, since it seems to be what is displayed to the user in case the condition fails.